spec/features/users_can_update_projects_spec.rb

```diff
@@ -1,20 +1,16 @@
 require "rails_helper"
 
 RSpec.feature "Users can reach the edit project page" do
+  include_context "Academies API returns an establishment"
+
+  let(:urn) { 12345 }
   let(:team_leader) { create(:user, :team_leader, email: "teamleader@education.gov.uk") }
   let(:delivery_officer) { create(:user, email: "user1@education.gov.uk") }
-  let(:establishment) { build(:academies_api_establishment) }
-  let(:establishment_result) { AcademiesApi::Client::Result.new(establishment, nil) }
-
-  before do
-    allow_any_instance_of(AcademiesApi::Client).to \
-      receive(:get_establishment) { establishment_result }
-  end
 
   context "the user is a team leader" do
     before(:each) do
       sign_in_with_user(team_leader)
-      @unassigned_project = Project.create!(urn: 1001)
+      @unassigned_project = Project.create!(urn:)
```

Comment:
This is too clever for me! :) Could we do `Project.create!(urn: urn)` to keep things nice and clear for simple folk like me?